modules/bootstrap/variables.tf

```diff
@@ -52,20 +52,48 @@ variable "files_md5" {
   Normally the map could be all empty, because all the files that exist before the `terraform apply` will have their hashes auto-calculated.
   This input is necessary only for the selected files which are created/modified within the same Terraform run as this module.
   The keys of the map should be identical with selected keys of the `files` input, while the values should be MD5 hashes of the contents of that file.
-  For example `{"dir/my.txt" = "6f7ce3191b50a58cc13e751a8f7ae3fd"}`
+
+  Example:
+  ```
+  files_md5 = {
+      "dir/my.txt" = "6f7ce3191b50a58cc13e751a8f7ae3fd"
+  }
+  ```
   EOF
   default     = {}
   type        = map(string)
 }
 
 variable "storage_share_name" {
-  description = "Name of storage share to be created that holds `files` for bootstrapping."
+  description = "Name of storage File Share to be created that holds `files` for bootstrapping."
   default     = "bootstrapshare"
   type        = string
 }
 
+variable "storage_share_quota" {
+  description = "Maximum size of a File Share."
+  default     = 50
+  type        = number
+}
+
+variable "storage_share_access_tier" {
+  description = "Access tier for the File Share."
+  default     = "Cool"
+  type        = string
+}
+
 variable "tags" {
-  description = "Azure tags to apply to the created Storage Account. A map, for example `{ team = \"NetAdmin\", costcenter = \"CIO42\" }`"
+  description = <<-EOF
+  A map of Azure tags to apply to the created Storage Account.
+
+  Example: 
+  ```
+  tags = {
+    team       = "NetAdmin"
+    costcenter = "CIO42"
```

Comment:
Perhaps match these tags to generic tags we use throughout the rest of the repo?

Reply:
I actually removed the example values. This is what we have in other modules. This way it matches a pattern.